configs/nafnet/README.md

```diff
@@ -0,0 +1,90 @@
+# NAFNet (ECCV'2022)
+
+> [Simple Baselines for Image Restoration](https://arxiv.org/abs/2204.04676)
+
+> **Task**: Image Restoration
+
+<!-- [ALGORITHM] -->
+
+## Abstract
+
+<!-- [ABSTRACT] -->
+
+Although there have been significant advances in the field of image restoration recently, the system complexity of the state-of-the-art (SOTA) methods is increasing as well, which may hinder the convenient analysis and comparison of methods. In this paper, we propose a simple baseline that exceeds the SOTA methods and is computationally efficient. To further simplify the baseline, we reveal that the nonlinear activation functions, e.g. Sigmoid, ReLU, GELU, Softmax, etc. are not necessary: they could be replaced by multiplication or removed. Thus, we derive a Nonlinear Activation Free Network, namely NAFNet, from the baseline. SOTA results are achieved on various challenging benchmarks, e.g. 33.69 dB PSNR on GoPro (for image deblurring), exceeding the previous SOTA 0.38 dB with only 8.4% of its computational costs; 40.30 dB PSNR on SIDD (for image denoising), exceeding the previous SOTA 0.28 dB with less than half of its computational costs.
+
+<!-- [IMAGE] -->
+
+<div align=center >
+ <img src="https://user-images.githubusercontent.com/43229734/199919292-81d307d9-144b-4d07-9f26-0c09d86e84a5.jpg" width="400"/>
+</div >
+
+## Results and models
+
+|                            Method                            | image size |    GoPro PSNR    |   GoPro SSIM   |    SIDD PSNR     |   SIDD SSIM    | GPU Info |                            Download                            |
```

Comment:
the head of the table can be PSNR or SSIM without annotation of dataset, which has been shown in configs.